platform/wab/src/wab/client/components/sidebar/token-controls.tsx

```diff
@@ -288,6 +312,21 @@ export const TokenFolderRow = observer(function TokenFolderRow(props: {
       }}
       groupSize={groupSize}
       isOpen={isOpen}
+      showActions={!readOnly}
+      actions={
+        !readOnly && (
+          <IconButton
+            onClick={async (e) => {
+              if (isOpen) {
+                e.stopPropagation();
+              }
+              await tokenControls.onAdd(tokenType, path);
```

Comment:
The modal for adding tokens automatically generates a new token name and highlights the input text, which could be a little inconvenient when adding to a folder/subfolder.